files/system/river/usr/bin/startriver

```diff
@@ -5,11 +5,12 @@
 # builds actually ran successfully without any errors!
 set -oue pipefail
 
-
+wayblue_origin_conf="/usr/etc/river/init"
 river_conf="$HOME/.config/river/init"
 
-if [ -f "$river_conf" ]; then
-    river
-else
-    XDG_CONFIG_HOME=/usr/etc/ river
+# Replace the homedir conf if it's missing or of wayblue origin
+if [ ! -f "$river_conf" ] || grep -q "ORIGIN: WAYBLUE" "$river_conf"; then
+    cp "$wayblue_origin_conf" "$river_conf"
```

Comment:
Since this runs in a `set -e` context, we should check writeability before we clobber so a user can't accidentally perm themselves out of a desktop session. `[ -w "$river_conf" ] && cp [...]`